standard/sections/clause_7_normative_text.adoc

```diff
@@ -25,19 +29,19 @@ The table below provides an overview of the set of properties that may be includ
 
 |``type``
 |**required**
-|A fixed value denoting the record as a GeoJSON ``Feature`` (see <<_validation>>)
```

Comment:
Overall comment: shouldn't we keep the `<< >>` markers?  They result in inline links for easy navigation.

Reply:
agreed. I removed this ONE, because it was linking to the validation in the ATS.